src/parameters.toml

```diff
@@ -2214,6 +2214,24 @@ value = 355
 type = "float"
 description = "Maximum temperature in the lookup table for optical properties in RRTMGP"
 
+# RCEMIP
+
+[SST_mean]
+value = 300
+type = "float"
+
+[SST_delta]
+value = 1.25
+type = "float"
+
+[SST_wavelength]
+value = 6000000
+type = "float"
+
+[SST_wavelength_latitude]
+value = 54
+type = "float"
+
```

Comment:
`description = "<some info> + source DOI"` against these params would be helpful. Once this is done we can merge this PR. Thanks!